<launch>

    <node pkg="integration_project" name="triangle_3d_points_teleop" type="triangle_3d_points_teleop" output="screen" />
    <node pkg="integration_project" name="triangle_area_resolver" type="triangle_area_resolver"/>
    
</launch>